.appveyor.yml

```diff
@@ -28,62 +28,62 @@ environment:
   matrix:
     - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
       GENERATOR: "MinGW Makefiles"
-      CMAKE_PREFIX_PATH: "C:\\MinGW"
+      cc_path: 'C:\MinGW\bin'
       BUILD_SHARED_LIBS: ON
 
     - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
       GENERATOR: "MinGW Makefiles"
-      CMAKE_PREFIX_PATH: "C:\\MinGW"
+      cc_path: 'C:\MinGW\bin'
       BUILD_SHARED_LIBS: OFF
 
     - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
       GENERATOR: "MinGW Makefiles"
-      CMAKE_PREFIX_PATH: "C:\\mingw-w64\\i686-5.3.0-posix-dwarf-rt_v4-rev0"
+      cc_path: "C:\mingw-w64\i686-5.3.0-posix-dwarf-rt_v4-rev0\mingw32\bin"
```

Comment:
This is yaml string, so backslash in double quotes needs to be escaped... `"C:\\mingw-"`

Reply:
This was fixed in the later commit: https://github.com/dlfcn-win32/dlfcn-win32/pull/48/commits/6fda813d16ea5f0881095b7fbf8b8d57a2813a39 .